rust/ql/lib/codeql/rust/elements/internal/OperationImpl.qll

```diff
@@ -140,7 +140,7 @@ module Impl {
   private class OperationMethodCall extends CallImpl::MethodCall instanceof Operation {
     OperationMethodCall() { super.isOverloaded(_, _, _) }
 
-    override Expr getPositionalArgument(int i) { result = super.getOperand(i + 1) }
+    override Expr getPositionalArgument(int i) { result = super.getOperand(i + 1) and i >= 0 }
```

Comment:
This does not actually matter in practice, since `TPositionalArgumentPosition` limits positions to be non-negative, however it still makes sense to fix.